### PR TITLE
Fix for login button being cut off on some mobile devices

### LINKED ISSUE #1706: Login- and signup-button are cutoff on Android phones with screen ratio 19.5:9
### Description

Both the login- and signup-button are cutoff and partially hidden from view. When scrolling the button is correctly rendered, but this is nonetheless a buggy experience for new users trying to create an account.

auth0_bug

### Environment

Please provide the following:

Lock version: 11.15

* Device: OnePlus A6013
* Browser version: Chrome 75.0.3770.143
* OS: Android 9

* Device: Huawei Mate 10
* Browser version: Chrome 75.0.3770.143
* OS: Android 9

### Reproduction

1. Navigate to development.equlitycheck.it.
2. Signup as a new user.
3. The Auth0-popup consistenly produces the bug for affected phones.

### PR DESCRIPTION
**Pull request by stevehobbsdev**
### Changes

This PR resolves an issue on mobile where the login button was being cut off towards the bottom of the screen. This PR implements the [technique on CSS Tricks](https://css-tricks.com/the-trick-to-viewport-units-on-mobile/) to dynamically calculate the screen height and apply that to a custom variable on `:root` where it can be picked up by the CSS. It uses `100vh` as a fallback for those browsers that do not support custom variables.

See screenshots in comments.

### References

Fixes #1706 
Partial fix for #1783 

### Testing

This has been tested manually on:

* Samsung Galaxy S6 (Chrome, Firefox)
* Samsung Galaxy S9 (Chrome, Firefox)
* Samsung Galaxy Note 8 (Chrome)
* Pixel 1 (Chrome)
* OnePlus 6T
* Chrome Desktop
* Firefox Desktop
* IE11

* [ ] This change adds unit test coverage
* [ ] This change adds integration test coverage
* [X] This change has been tested on the latest version of the platform/language

### Checklist

* [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
* [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
* [x] All existing and new tests complete without errors
* [x] All code quality tools/guidelines have been run/followed
* [ ] All relevant assets have been compiled
* [ ] All active GitHub checks have passed

### CODE REVIEW

**Comment by stevehobbsdev** [comment]
**Galaxy Note 8**

galaxy-note-8

**Comment by stevehobbsdev** [comment]
**Galaxy S6 (Chrome)**

galaxy-s6-chrome

**Comment by stevehobbsdev** [comment]
**Galaxy S9 (Chrome)**

galaxy-s9-chrome

**Comment by stevehobbsdev** [comment]
**OnePlus 6T (Chrome)**

oneplus-6t-chrome

**Comment by stevehobbsdev** [comment]
**Pixel 1 (Chrome)**

pixel-1-chrome

File: css/index.styl [hunk 1]
```diff
@@ -1048,7 +1049,8 @@ loadingSize = 30px
     .auth0-lock-widget-container
       //mobile view
       @media screen and (max-width: 480px)
-        height 100%
+        height 100vh  /* Support browsers where custom properties are not supported */
```
Comment (joshcanhelp) [inline css/index.styl]: I'm not very familiar with `styl` files but I would guess that this could be a mixin or something to avoid duplication?

Comment (stevehobbsdev) [reply css/index.styl]: Yes it can! Thanks for the idea. I've moved it into a mixin now.

**Pull request merged**